ocs_ci/ocs/resources/mcg.py

```diff
@@ -739,6 +736,33 @@ def cli_create_bucketclass(
                 f"bucketclass create {placement_type}{name}{backingstores}{placement_policy}{replication_policy}"
             )
 
+    def cli_create_bucketclass_over_namespacestores(
+        self,
+        name,
+        namespacestores,
+        namespace_policy,
+    ):
+        """
+        Creates a new NooBaa bucket class using the noobaa cli over namespace stores
+        Args:
+            name (str): The name to be given to the bucket class
+            namespacestores (list): The namespaces stores to use as part of the policy
+            namespace_policy (dict): The namespace policy to be used
+
+        Returns:
+            OCS: The bucket class resource
+
+        """
+        namestores_name_list = [
+            namespacestore.name for namespacestore in namespacestores
+        ]
+        namestores_name_str = f"{','.join(namestores_name_list)}"
+        namespace_policy_type = namespace_policy["type"].lower()
+        self.exec_mcg_cmd(
+            f"bucketclass create namespace-bucketclass {namespace_policy_type} "
+            f"--resource={namestores_name_str} {name}"
+        )
```

Comment:
This is correct for the `single` NSS type, but please also address the `multi` and `cache` options. 

For the scope of this PR, just raise a `NotImplementedError` for the other two types. We'll implement those in follow-up PRs.

Reply:
@sagihirshfeld Done.